src/lib.rs

```diff
@@ -83,54 +92,104 @@ mod window_counter;
 pub mod connectors;
 pub mod resolvers;
 
+/// Uniquely identifies a claim
+#[derive(Copy, Clone, Debug)]
+pub(crate) struct ClaimId(pub u64);
+
+impl ClaimId {
+    fn new() -> Self {
+        let id = usdt::UniqueId::new().as_u64();
+        Self(id)
+    }
+}
+
+impl Default for ClaimId {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
 /// USDT probes for tracing how qorb makes pools and hands out claims.
 #[cfg(feature = "probes")]
 #[usdt::provider(provider = "qorb")]
 mod probes {
     /// Fires right before attempting to acquire a claim from the pool.
-    fn claim__start(id: &usdt::UniqueId) {}
+    fn claim__start(pool: &str, claim_id: u64) {}
 
     /// Fires when a claim is successfully acquired from the pool.
-    fn claim__done(id: &usdt::UniqueId) {}
+    ///
+    /// Also identifies the underlying slot which is being used.
```

Comment:
Is the relationship: Pools contain one slot set for each backend?  Do we need information about the set here?